confectory-core/src/test/java/net/obvj/confectory/ConfectoryTest.java

```diff
@@ -0,0 +1,50 @@
+package net.obvj.confectory;
+
+import static net.obvj.junit.utils.matchers.AdvancedMatchers.instantiationNotAllowed;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import org.junit.Test;
+
+import net.obvj.confectory.settings.ConfectorySettings;
+
+/**
+ * Unit tests for the {@link Confectory} class.
+ * 
+ * @author FernandoNSC (Fernando Tiannamen)
+ */
+public class ConfectoryTest
+{
+
+    ConfigurationContainer configurationContainer = new ConfigurationContainer();
+    
+    @Test
+    public void constructor_instantiationNotAllowed()
+    {
+        assertThat(Confectory.class, instantiationNotAllowed());
+    }
+
+    @Test
+    public void ensure_global_default_configuration_exists()
+    {
+        assertNotNull(Confectory.container());
+    }
+
+    @Test
+    public void ensure_facade_dataFetchStrategy_and_defaultNullValueProvider()
```

Comment:
(1) Please use the following naming convention for JUnit test names:
`method_testCondition_expectedResult()`
or
`method_expectedResult()` (when no condition or params apply)
For example:
`settings_confectorySettingsInstance` (here it's OK not to have the condition because the method has no params.

(2) It's not a good idea to rely on the internal attributes of the ConfectorySettings class, because they can change and we don't want this Façade test to be changed by every modification in ConfectorySettings.
So, checking that `Confectory.settings()` returns exactly the same object as `ConfectorySettings.getInstance()` is the best way to test this unit in particular.